Dotnet/Friday.RockPaperScissors/README.md

```diff
@@ -0,0 +1,13 @@
+## How to Run:
+
+### Run with arguments
+
+```
+dotnet run -- args1 args2
```

Comment:
When I try to run the project like this it does not work. But it does work if I provide the path to the `.dll`.

```bash
cd Dotnet/Friday.RockPaperScissors

dotnet src/Friday.RockPaperScissors/bin/.../Friday.RockPaperScissors.dll -- args1 args2
```

Is this an the expected behaviour? May this be some issue on my side related to some configuration?

Reply:
@joaorbrandao the dotnet run applies in the directory where the file *.csproj is located

In this case, is: `dojo-rock-paper-scissors/Dotnet/Friday.RockPaperScissors/src/Friday.RockPaperScissors/`

Reply:
Thanks for clarifying @francisco25friday, that works as expected!